packages/web/src/components/range/RangeInput.js

```diff
@@ -21,11 +21,14 @@ class RangeInput extends Component {
 			isStartValid: true,
 			isEndValid: true,
 		};
+
+		this.startInputRef = React.createRef();
+		this.endInputRef = React.createRef();
 	}
 
 	componentDidUpdate(prevProps) {
-		if (!isEqual(this.props.value, prevProps.value)) {
-			this.handleSlider(this.props.value);
+		if (!isEqual(this.props.defaultValue, prevProps.defaultValue)) {
```

Comment:
We should only watch for `value` prop updates. `defaultValue` should only be used on initial mount.